event/src/unvalidated/event.rs

```diff
@@ -9,7 +9,46 @@ use std::{collections::HashMap, fmt::Debug};
 use tokio::io::AsyncRead;
 use tracing::debug;
 
-use super::{cid_from_dag_cbor, init, signed};
+use super::{cid_from_dag_cbor, init, signed, Payload};
+
+/// Helper function for Event::decode_car for gathering all the Ipld blocks used by a time event
```

Comment:
no functional change here, just moving it out the impl for Event

Reply:
should we move it to be an impl on `RawTimeEvent`? could be `pub(crate)` if we don't want to make it pub yet

Reply:
given that it would still need to take the proof and car_blocks as arguments, I actually feel like it makes more sense to leave it as a floating helper function since it really is pretty specific to the context it's being used in in decode_car, at least for now.